Resources/Prototypes/GameRules/meteorswarms.yml

```diff
@@ -87,6 +87,12 @@
   - type: NextEvent # DeltaV: Queue Event
   - type: GameRule
   - type: RampingStationEventScheduler
+    timeKeyPoints: # DeltaV
+      - 0, 15
+      - 30, 10
+      - 60, 7
+      - 90, 5
+      - 120, 3
```

Comment:
unindent them by 2 spaces, also this is too extreme as average 3 minutes between meteors prevents people extending rounds